src/test/scala/chiseltest/coverage/SimulatorCoverageTest.scala

```diff
@@ -24,10 +24,10 @@ abstract class SimulatorCoverageTest(name: String, backend: SimulatorAnnotation,
     val cov = getCoverage(r)
 
     // the cover point in the main module are not prefixed
-    assert(cov.keys.toList == List("SIM"))
+    assert(cov.keys.toList == List("user_cov"))
 
     // since we executed one step and all inputs are zero by default, we expect the count to be 3
```

Comment:
I know this was not part of your change but this comment is either wrong or confusing.
```suggestion
    // since we executed one step and all inputs are zero by default, we expect the count to be 1
```